tests/cli/upgrade_helpers/test_upgrade_helper_v11.py

```diff
@@ -0,0 +1,589 @@
+        data_docs/
+            local_site/
+                expectations/
+                    .gitkeep
+                static/
+                    .gitkeep
+                validations/
+                    diabetic_data/
+                        warning/
+                            20200430T191246.763896Z/
+                                c3b4c5df224fef4b1a056a0f3b93aba5.html
+        validations/
+            diabetic_data/
+                warning/
+                    20200430T191246.763896Z/
+                        c3b4c5df224fef4b1a056a0f3b93aba5.json
+"""
+    obs_project_tree_str = gen_directory_tree_str(v10_project_directory)
+    assert obs_project_tree_str == expected_project_tree_str
+
+
+@freeze_time("09/26/2019 13:42:41")
+def test_basic_project_upgrade(v10_project_directory, caplog):
+    # test project upgrade that requires no manual steps
+
+    runner = CliRunner(mix_stderr=False)
+    result = runner.invoke(
+        cli,
+        ["project", "upgrade", "-d", v10_project_directory],
+        input="\n",
+        catch_exceptions=False,
+    )
+    stdout = result.stdout
+
+    assert "Checking project..." in stdout
+    assert (
+        "Your project appears to have an out-of-date config version (1.0) - the version number must be at least 2."
+        in stdout
+    )
+    assert (
+        "Would you like to run the Upgrade Helper to bring your project up-to-date? [Y/n]:"
+        in stdout
+    )
+    assert (
+        """\
+UpgradeHelperV11 will upgrade your project to be compatible with Great Expectations 0.11.x.
+
+**WARNING**: Before proceeding, please make sure you have appropriate backups of your project.
+
+Automated Steps
+================
+
+The following Stores and/or Data Docs sites will be upgraded:
+
+    - Validation Stores: validations_store
+    - Data Docs Sites: local_site
+
+Manual Steps
+=============
+
+No manual upgrade steps are required.
+
+Upgrade Confirmation
+=====================
+
+Please consult the 0.11.x migration guide for instructions on how to complete any required manual steps or
+to learn more about the automated upgrade process:
+
+    https://docs.greatexpectations.io/en/latest/how_to_guides/migrating_versions.html#id1
+
+Would you like to proceed with the project upgrade? [Y/n]:
+
+Upgrading project...\
+"""
+        in stdout
+    )
+    assert (
+        "Your project was successfully upgraded to be compatible with Great Expectations 0.11.x."
+        in stdout
+    )
+    assert (
+        "The config_version of your great_expectations.yml has been automatically incremented to 2.0."
+        in stdout
+    )
+    assert (
+        f"""\
+A log detailing the upgrade can be found here:
+
+    - {v10_project_directory}/uncommitted/logs/project_upgrades/UpgradeHelperV11_20190926T134241.000000Z.json[0m[0m
+
+================================================================================
+[0m
+[32mUpgrade complete. Exiting...\
+"""
+        in stdout
+    )
```

Comment:
This is more of a "I'm curious about.." question because I'm sure the tests are passing.  Where do the [0m [32m messages come from?